lib/config/autoconfig.js

```diff
@@ -312,8 +313,11 @@ Registry.prototype = {
  */
 function extendFromRecommended(config) {
     var newConfig = lodash.assign({}, config);
+
+    ConfigOps.normalizeToStrings(newConfig);
+
     var recRules = Object.keys(recConfig.rules).filter(function(ruleId) {
-        return (recConfig.rules[ruleId] === 2 || recConfig.rules[ruleId][0] === 2);
+        return ConfigOps.isErrorSeverity(recConfig.rules[ruleId]);
```

Comment:
I may have preferred normalizing to numbers and having a `string -> number` mapping somewhere but I guess this is also okay.